src/Files.App/Helpers/NavigationHelpers.cs

```diff
@@ -36,6 +37,19 @@ public static Task<bool> OpenTabInNewWindowAsync(string tabArgs)
 			return Launcher.LaunchUriAsync(folderUri).AsTask();
 		}
 
+		public static void OpenInSecondaryPane(IShellPage associatedInstance, ListedItem listedItem)
+		{
+			if(associatedInstance is null || listedItem is null)
+				return;
+
+			IUserSettingsService userSettingsService = Ioc.Default.GetRequiredService<IUserSettingsService>();
+			if (!userSettingsService.PreferencesSettingsService.IsDualPaneEnabled)
+				return;
+
+			if (listedItem is not null)
+				associatedInstance.PaneHolder?.OpenPathInNewPane((listedItem as ShortcutItem)?.TargetPath ?? listedItem.ItemPath);
+		}
+
```

Comment:
Why here `listedItem` is null checked twice?
I think you should also change `IUserSettingsService` to `var`

Reply:
Good catch, looks like it can be removed